lale/lib/sklearn/logistic_regression.py

```diff
@@ -61,7 +64,10 @@ def predict_proba(self, X):
     'X': {
       'description': 'Features; the outer array is over samples.',
       'type': 'array',
-      'items': {'type': 'array', 'items': {'type': 'number'}}}}}
+      'items': {'anyOf':
+          [{'type': 'array', 'items': {'type': 'number'}},
+           {'type': 'number'}
+          ]}}}}
```

Comment:
Similarly for predict: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression.predict